specs/meta/validator-deposit.md

```diff
@@ -0,0 +1,78 @@
+<!-- DOCTOC SKIP -->
+
+# Validator Deposit Guide
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Deposit into `ValidatorPool`](#deposit-into-validatorpool)
+- [Withdraw from `ValidatorPool`](#withdraw-from-validatorpool)
+- [Try unbond in `ValidatorPool`](#try-unbond-in-validatorpool)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+We want the [validator](../../components/validator/) role to be decentralized. Like how the PoS mechanism works, to
+achieve this, the validator needs to bond ETH at every
+[output submission](../validations.md#submitting-l2-output-commitments). A Validator can deposit into the contract
+`ValidatorPool` at once for
+convenience. This guide teaches you how to deposit, withdraw, and try to unbond in `ValidatorPool`.
```

Comment:
The `deposit` is not for convenience. It is required to bond ETH for output submission.
I think it would be better to describe two steps of deposit and bond specifically.
Also, we need to mention that the bonded ETH will be unbonded after the finalization of the output.

Reply:
I've added a brief explanation as you said. Since this is a guide to utilizing the commands, a more detailed explanation should probably be included in `validations.md`.

Reply:
Oh, right. The detailed description should be in `validations.md` or `validators.md`.